apl-desktop/src/main/java/com/apollocurrency/aplwallet/apldesktop/DesktopApplication.java

```diff
@@ -454,7 +447,7 @@ public void startDesktopApplication() {
             mainStage.show();
```

Comment:
Possible NPE, it I'm not mistaken...

Reply:
Its not possible, because DesktopApplication.start initialize it with primaryStage